evap/staff/templates/staff_user_list.html

```diff
@@ -0,0 +1,105 @@
+                        {% trans 'Hide' %}
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="input-group col-3">
+            <input type="search" name="search" class="form-control" placeholder="{% trans 'Search...' %}" />
+            <div class="input-group-append">
+                <button class="btn btn-light text-secondary" type="button" data-reset="search" data-toggle="tooltip" data-placement="top" title="{% trans 'Clear search filter' %}">
+                    <span class="fas fa-backspace"></span>
+                </button>
+              </div>
+        </div>
+    </div>
+
+    <div class="card mb-3">
+        <div class="card-body">
+            <table class="table table-striped table-vertically-aligned user-table">
+                <thead>
+                    <tr>
+                        <th style="width: 25%">{% trans 'Name' %}</th>
+                        <th style="width: 35%">{% trans 'Email' %}</th>
+                        <th style="width: 30%">{% trans 'Information' %}</th>
+                        <th style="width: 10%">{% trans 'Actions' %}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                {% for user in users %}
+                    <tr id="user-row-{{ user.id }}">
+                        <td>{{ user.full_name }}</td>
+                        <td>{% if user.email %}{{ user.email }}{% endif %}</td>
+                        <td>{% include 'staff_user_badges.html' with user=user %}</td>
+                        <td>
+                            <a href="{% url 'staff:user_edit' user.id request.path %}" class="btn btn-secondary btn-sm"><span class="fas fa-pencil-alt"></span></a>
+                            {% if user.can_be_deleted_by_manager %}
+                                <button type="button" class="btn btn-danger btn-sm" onclick="deleteModalShow({{ user.id }}, '{{ user.full_name|escapejs }}');">
+                                    <span class="fas fa-trash"></span
+                                ></button>
+                            {% else %}
+                                <button type="button" disabled class="btn btn-sm btn-danger" data-toggle="tooltip" data-placement="left"
+                                    title="{% blocktrans %}This user contributes to an evaluation, participates in an evaluation whose participations 
+                                    haven't been archived yet or has special rights and as such cannot be deleted.{% endblocktrans %}">
+                                    <span class="fas fa-trash"></span>
+                                </button>
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    </div>
+{% endblock %}
+
+{% block modals %}
+    {{ block.super }}
+    {% trans 'Delete user' as title %}
```

Comment:
@janno42 do we want to keep the delete button here?

Reply:
no, please remove the delete button. as specified, the delete button should only be available in the user form to further increase the page loading time for the user list.